pages/applications.tsx

```diff
@@ -0,0 +1,192 @@
+// Copyright (C) 2023 Nethesis S.r.l.
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import type { NextPage } from 'next'
+import { useTranslation } from 'react-i18next'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faArrowRight,
+  faPhoneVolume,
+  faCameraSecurity,
+  faVideo,
+  faChartSimple,
+  faArrowUpRightFromSquare,
+} from '@nethesis/nethesis-solid-svg-icons'
+import { Button } from '../components/common'
+import { faPhone } from '@fortawesome/free-solid-svg-icons'
+import { getApiEndpoint } from '../lib/utils'
+import { getApiScheme } from '../lib/utils'
+
+const Applications: NextPage = () => {
+  const { t } = useTranslation()
+  const pathCompany = getApiEndpoint()
+  const apiScheme = getApiScheme()
+  const linkNethvoiceReport = apiScheme + pathCompany + '/pbx-report/'
+
+  return (
+    <>
+      <div>
+        <h1 className='text-2xl font-semibold mb-4 text-gray-900 dark:text-gray-100'>
+          {t('Applications.Applications')}
+        </h1>
+        <div>
+          <span className='text-md font-normal mb-4 text-gray-600 dark:text-gray-100'>
+            {t('Applications.Internal')}
+          </span>
+          <div className='flex flex-wrap gap-5'>
+            <div className='max-w-sm mt-5 bg-white border-gray-200 rounded-lg shadow dark:bg-gray-900'>
+              <div className='bg-emerald-50 rounded-lg flex flex-col justify-center items-center'>
+                <span className='h-10 w-10 overflow-hidden rounded-full bg-emerald-600 dark:bg-emerald-500 flex relative top-[-1.25rem]'>
+                  <FontAwesomeIcon
+                    icon={faPhoneVolume}
+                    className='h-4 w-4 m-3 rounded-lg text-white'
+                    aria-hidden='true'
+                  />
+                </span>
+                <div className='pb-1 rounded-lg'>
+                  <h5 className='flex mb-2 font-medium  text-gray-700'>
+                    {t('Applications.Telephone lines and announcements')}
+                  </h5>
+                </div>
+              </div>
+              <div className='bg-white dark:bg-gray-900 p-5 px-12 rounded-lg flex flex-col justify-center'>
+               <div className='mb-8 font-normal text-xs justify-center items-center text-gray-600 dark:text-gray-400'>
+                <p className='text-center'>
+                  {t(
+                    'Applications.Manage your phone lines, activate announcements, voicemail or forward calls',
+                  )}
+                </p>
+               </div>
+                <div className='text-gray-900 dark:text-gray-100 text-sm flex items-center justify-center'>
+                  <span className='font-semibold'>Coming soon..</span>
```

Comment:
Let's add a i18n string for "Coming soon". Also, I think it has too much contrast since it's not even clickable. How about `text-gray-500`?